advocacy_docs/edb-postgres-ai/ai-accelerator/limitations.mdx

```diff
@@ -4,38 +4,38 @@ navTitle: Limitations
 description: Limitations of the EDB Postgres AI - AI Accelerator Pipelines.
```

Comment:
Flagging this description for consistency of how we might refer to this product in the descriptions.